cellect_env.rb

```diff
@@ -83,8 +83,8 @@ def database_url
     "postgresql://#{@pg_user}:#{@pg_pass}@#{@pg_host}:#{@pg_port}/#{@pg_db}?pool=#{connection_pool_value}"
   end
 
-  def zk_url
-    ENV['ZK_URL'] ||
-    "#{ENV["ZK_PORT_2181_TCP_ADDR"]}:#{ENV["ZK_PORT_2181_TCP_PORT"]}"
+  def redis_url
+    ENV['ATTENTION_REDIS_URL'] ||
+    "redis://#{ENV["REDIS_PORT_6379_TCP_ADDR"]}:#{ENV["REDIS_PORT_6379_TCP_PORT"]}/0"
```

Comment:
perhaps move the docker_compose ENV vars to a failed yaml lookup in the `load_redis_yaml` and then mimic the `database_url` method to use the yaml + fallback configs? this way we get an order of 1. explicit ENV var, 2. yaml config, 3. docker_compose defaults.